learn/best_practices/integer_division.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: book
+title: Integer Division
+permalink: /learn/best_practices/integer_division
+---
+
+Just like in Python 2.x or C, when doing things like `(N-1)/N` where `N`
+is an integer and you want a floating point division, force the compiler
+to use floats at the right hand side, for example by:
+
+``` fortran
+(N - 1.0_dp)/N
```

Comment:
There was no RHS, since there was no LHS
```suggestion
res = (N - 1.0_dp)/N
```

Reply:
Should we declare the LHS as `real(dp) :: res` as well here?